src/test/java/com/PlaceLab/Amila/qamp/tests/SmokeTest.java

```diff
@@ -0,0 +1,37 @@
+package com.PlaceLab.Amila.qamp.tests;
+
+import com.PlaceLab.Amila.qamp.utils.WebDriverSetup;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
```

Comment:
see if you have included some imports you don't use, if you did - they can be removed. It's just a good practice to not include the unused imports, you can configure your Intellij to remove automatically